py-polars/polars/functions/as_datatype.py

```diff
@@ -185,6 +185,7 @@ def duration(
     minutes: Expr | str | int | None = None,
     hours: Expr | str | int | None = None,
     weeks: Expr | str | int | None = None,
+    time_unit: TimeUnit = "ns",
```

Comment:
"ns" is a bit of an odd default, isn't it usually "us" in Polars?

I'm setting it as "ns" here to preserve backwards compatibility, but (separately) it may be worth considering "us" as the default? this would also reduce the likelihood of overflows like the one reported in the linked issue

Reply:
Yes, it is. We should set this to `us`. It is a bu fix if we didn't.